test/Atc.Test.Tests/MemberAutoNSubstituteDataAttributeTests.cs

```diff
@@ -25,4 +25,29 @@ public class MemberAutoNSubstituteDataAttributeTests
         dependantType.Should().NotBeNull();
         dependantType.Dependency.Should().Be(interfaceType);
     }
+
+    public static readonly IEnumerable<object[]> TestDataMultiple = new[]
+    {
+        new object[] { SampleEnum.One, SampleEnum.Two },
+        new object[] { SampleEnum.One, SampleEnum.One },
+    };
+
+    [Theory]
+    [MemberAutoNSubstituteData(nameof(TestDataMultiple))]
+    public void MemberAutoNSubstituteData_Should_Call_For_Each_Value_Mutliple(
```

Comment:
MemberAutoNSubstituteData_Should_Call_For_Each_Value_Mutliple -> Multiple.

Besides that i dont understand the test name?!